tests/WE2E/setup_WE2E_tests.sh

```diff
@@ -62,13 +62,13 @@ EXPTS_DIR=${TOP_DIR}/expt_dirs
 # Set the path to the machine-specific test suite file.
 #-----------------------------------------------------------------------
 
-auto_file=${scrfunc_dir}/machine_suites/${machine}.txt
+auto_file=${scrfunc_dir}/machine_suites/fundamental.txt
```

Comment:
We do want this to still be a test suite for the machine.

Reply:
I thought it would be easier to have one file for fundamental tests that should be same on all machines. (`hera.txt` and `jet.txt` were identical when i removed them). Otherwise I would have to create `orion.txt`, `gaea.txt` etc.

Reply:
I think i found a good compromise for this. If a machine specific list of test cases is available such as `hera.txt` or `jet.txt`, that will be used, otherwise we fall back on to `fundamental.txt`. That way we can remove/add a test case for specific machines.